modules/contributing-quick-starts.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * web_console/creating-quick-start-tutorials.adoc
+
+[id="contributing-quick-starts_{context}"]
+= Contributing quick starts
+
+In {product-title} 4.7, a quick start custom resource is added. This allows Operators and administrators to contribute new quick starts to the cluster.
+
+.Prerequisites
+
+* You must have cluster administrator privileges.
+
+.Procedure
+
+. To create a new quick start, run:
++
+----
+$ oc create -f my-quick-start.yaml
```

Comment:
This step isn't actually creating anything since we haven't told them to create a YAML spec yet. I'd suggest adding a step before this for something like:

```
$ oc get -o yaml consolequickstart spring-with-s2i > my-quick-start.yaml
```

or whatever an approved option is, so that they at least have a full example spec to play with and modify.